custom-routes-vis/src/main/webapp/index.html

```diff
@@ -41,18 +41,18 @@ <h5>Destination Coordinates:</h5>
           <option value="compute-routes">ComputeRoutes API(V1)</option>
           <option value="compute-routes-alpha">ComputeCustomRoutes API(V1 alpha)</option>
         </select>
-        <input id="api-key" class="form-control" placeholder="API Key (Defaults to use Directions API Key)"></input>
+        <input id="api-key" class="form-control" placeholder="API Key (Optional, Default Key Exists)"></input>
         <div id="rate-card-data" class="hidden">
           <div class="row">
             <label for="cost-per-minute" class="col-sm-5">Cost Per Minute:</label>
             <div class="col-sm-7">
-              <input type="number" id="cost-per-minute" class="form-control"></input>
+              <input type="number" id="cost-per-minute" class="form-control" placeholder="Required"></input>
```

Comment:
This is actually not required. None of the rate card fields are required, so may changed this to also say optional.

Reply:
This will involve changing the logic. TODO later

Reply:
I checked with the curl command and the rate card is actually required for the custom routes API; otherwise it returns an error saying "route_objective must be specified." @tzhong92 Can you confirm this?
So I think we should mark those as required.

Reply:
RateCard is a required field but the fields(costPerMinute, costPerKm, includeTolls) inside RateCard are optional. See https://developers.google.com/maps/documentation/routes_preferred/reference/rest/v1alpha/TopLevel/computeCustomRoutes?authuser=0#RouteObjective

Reply:
I get this from the curl when I leave empty both costPerMin and costPerKm: "message": "At least one of cost_per_minute or cost_per_km must be specified." I guess I can mark them as optional but put the message at the bottom.